go.mod

```diff
@@ -44,3 +44,6 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+// NOTE: Temporary for testing purposes
+replace github.com/gophercloud/gophercloud/v2 => github.com/Sharpz7/gophercloud/v2 v2.0.0-20240719114351-6815cc5264b6
```

Comment:
I really struggled with this. I need to replace the v1 version as well, but get a bunch of issues.

```bash
go: downloading github.com/Sharpz7/gophercloud v1.0.0-20240719114351-6815cc5264b6
go: github.com/gophercloud/utils@v0.0.0-20230523080330-de873b9cf00d requires
        github.com/gophercloud/gophercloud@v1.3.0 (replaced by github.com/Sharpz7/gophercloud@v1.0.0-20240719114351-6815cc5264b6): version "v1.0.0-20240719114351-6815cc5264b6" invalid: go.mod has post-v1 module path "github.com/gophercloud/gophercloud/v2" at revision 6815cc5264b6
```

```golang
replace github.com/gophercloud/gophercloud/v2 => github.com/Sharpz7/gophercloud/v2 v2.0.0-20240719114351-6815cc5264b6

replace github.com/gophercloud/gophercloud => github.com/Sharpz7/gophercloud v1.0.0-20240719114351-6815cc5264b6
```